packages/core/jest.config.js

```diff
@@ -18,4 +19,5 @@ module.exports = {
         '^Services$': '<rootDir>/src/Services/index.js',
         '^Stores/(.*)$': '<rootDir>/src/Stores/$1',
     },
+    transformIgnorePatterns: ['/node_modules/(?!(@deriv-com/quill-ui|@simplewebauthn/browser)).+\\.js$'],
```

Comment:
Comment from Bala:
can we add @deriv-com/quill-ui in the jest.config.base.js so that it's applied for all the packages?

Reply:
When I moved it jest.config.base tests are failing again. It's weir cause we are not rewrite it anywhere
<img width="1639" alt="Screenshot 2024-06-21 at 11 58 10 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/09e8444f-4f2b-478c-b043-90683b2eb7d8">

Reply:
![image](https://github.com/binary-com/deriv-app/assets/56330681/e1c86614-db65-4484-a9b9-20ee4d58c73e)

Ideally `transformIgnorePatterns` should be picked from the `baseConfigForPackages`. Did we override by any chance in the core package?

Reply:
I moved transformIgnorePatterns to baseConfigForPackages (in jest.config.base.js) and it didn't help. Also tried to add to jest.config.js (separately and together with baseConfigForPackages), but it didn't work either(
Checked places where we could possible override it, but didn't find :/

Reply:
Okay. It's fine then.

Reply:
Thanks for checking